Main.sublime-menu

```diff
@@ -13,19 +13,11 @@
                 [
                     {
                         "caption": "PackageResourceViewer",
-                        "children":
-                        [
-                            {
-                                "command": "open_file",
-                                "args": {"file": "${packages}/PackageResourceViewer/PackageResourceViewer.sublime-settings"},
-                                "caption": "Settings – Default"
-                            },
-                            {
-                                "command": "open_file",
-                                "args": {"file": "${packages}/User/PackageResourceViewer.sublime-settings"},
-                                "caption": "Settings – User"
-                            }
-                        ]
+                        "command": "edit_settings",
```

Comment:
Yes, I'll create a separate branch for ST2. My point was just that what I have is stable (from an ST2 perspective), so not expecting  many bugs to come up with it. :) All the plugins I have right now are compatible with both ST2 and ST3, so I'll just need to read through the package control docs on how to do a version for each.